src/app/_mocked/mock-bar.component.ts

```diff
@@ -9,9 +9,9 @@ import { ColourSeriesData } from '../_models';
 export class MockBarComponent {
   readonly maxNumberBars = 50;
 
-  constructor() {}
-
-  browserOnly(_: () => void): void {}
+  browserOnly(_: () => void): void {
+    console.log('MockBarComponent.browserOnly');
```

Comment:
Did you forget this? Or is it being used to debug something, and it should remain ?

Reply:
The console.log statements were there to prevent the linter from complaining about empty functions, so I replaced them with comments (which also have this effect) in the relevant mock files